### ISSUE TITLE
Set 'offline_access' scope but token still only hour long

### ISSUE THREAD

**Issue by jack-quartic** [O365/python-o365#879]
Hi all,

I've got my app registered in Azure with the following _delegated permissions_:
- Mail.Read
- Mail.ReadWrite
- MailSend
- offline_access
- User.Read

When I authenticate my app, I use these scopes in the account.authenticate(scopes="_____"): and after feeding back the re-direct URL,
I checked my token text file to check when it expires, and it still says an hour. The token text file also indicates scopes that I do not have set (image below), so I'm just a little confused, and I'm sure I'm missing something (code is below). Any help is appreciated.

```
import O365
from O365 import MSGraphProtocol

credentials = ('client ID', 'client secret')
protocol = MSGraphProtocol(api_version='beta')
acct = O365.Account(credentials, protocol=protocol)

if acct.authenticate(scopes=["https://graph.microsoft.com/Mail.Read","https://graph.microsoft.com/Mail.ReadWrite",
                             "https://graph.microsoft.com/Mail.Send","https://graph.microsoft.com/offline_access",
                             "https://graph.microsoft.com/User.Read"
                            ]):
    print("Authenticated!")
```
![auth_token_text_file](https://user-images.githubusercontent.com/107424767/202592795-2c5fa078-e9c8-459f-a153-5ce4b3a05454.png)

**Comment by janscas** [comment]
Its just "offline_access" withut the url

**Comment by jack-quartic** [comment]
I gave that a try, but my token file still indicates only an hour of validity. The pypi.org documentation mentions that I could use the scope helper 'basic' which covers 'offline_access'. I likewise tried that to no avail.
![image](https://user-images.githubusercontent.com/107424767/202763295-3fa992d0-65ba-44b4-9e49-045582fbfbd6.png)

**Comment by janscas** [comment]
> I gave that a try, but my token file still indicates only an hour of validity. The pypi.org documentation mentions that I could use the scope helper 'basic' which covers 'offline_access'. I likewise tried that to no avail.
> ![image](https://user-images.githubusercontent.com/107424767/202763295-3fa992d0-65ba-44b4-9e49-045582fbfbd6.png)

You can, but that’s not what you set as scopes in your code

**Comment by jack-quartic** [comment]
After adding the 'offline_access' without the rest of the URL, as you suggested, it seems to be working. After authenticating, the UNIX expiration time in the token text file still only says 1 hour, but I waited more than an hour, and then tried accessing the token and sending email, and it worked. And using .refresh_token() works now that I added the 'offline_access' scope, so I think I'm all good. Thanks!
